### PR TITLE
Populating missing ParticipantProfile::NPQ records

### PR DESCRIPTION
**Pull request by BroiSatse**
ParticipantProfile::NPQ is now shadowing the NPQProfile. This PR introduced a rake task to migrate the create the remaining ParticipantProfile::NPQ.

### CODE REVIEW

File: lib/tasks/data.rake [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace :data do
+  task populate_npq_profiles: :environment do
+    NPQProfile.where.not(id: ParticipantProfile::NPQ.select(:id)).find_each do |profile|
+      profile.send(:update_participant_profile)
```
Comment (kicferk1) [inline lib/tasks/data.rake:4]: Why `.send(:update_participant_profile)` rather than `.update_participant_profile`?

Comment (BroiSatse) [reply lib/tasks/data.rake:4]: It is a private method. I could have made it public, but it is to go away very soon together with this task :P

File: spec/cypress/support/axe.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ Cypress.Commands.overwrite("injectAxe", () => {
   // https://github.com/component-driven/cypress-axe/issues/6#issuecomment-726986454
   cy.window({ log: false }).then((win) => {
     // eslint-disable-next-line global-require
-    const axe = require("axe-core/axe.js");
+    const axe = require("axe-core/axe");
```
Comment (kicferk1) [inline spec/cypress/support/axe.js:9]: Why do we need this change?

Comment (BroiSatse) [reply spec/cypress/support/axe.js:9]: JS lint does not like extensions being included in `require`. I have no clue why it was complaining about this on this PR...

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**